File: PO1508/python/Principal.py
from util import Util
import time

import threading
from ordenacao_Bolha import Ordenacao



lista = []
lista_bolha = []

Util.gerar_numeros_lista(lista, 10000, 10000)
Util.gerar_numeros_lista(lista_bolha, 10000, 10000)

threading.Thread(
    target=lambda:(
        #Marcando tempo de ordenação 
        #inicio = time.time(),

        print("Ordenando a lista de numeros em bolha ..."),

        #ordenando a lista por bolha....
        Ordenacao.bolha(lista_bolha),

       # fim = time.time(),
       # tempo_decorrido = fim - inicio,

        #print(f"Tempo decorrido: {tempo_decorrido:.6f} segundos"),


        #Util.exibir(lista_palavras, "Lista de palavras ordenadas")
        print("Feita a ordenacao em bolha..;"),
    )
).start()

threading.Thread(
    target=lambda: (
        #Marcando tempo de ordenação 
        #inicio = time.time(),

        print("Ordenando a lista de numeros com o sort..."),

        #ordenando a lista....
        lista.sort(),

        #fim = time.time(),
        #tempo_decorrido = fim - inicio,

        #print(f"Tempo decorrido: {tempo_decorrido:.6f} segundos"),


        #Util.exibir(lista_palavras, "Lista de palavras ordenadas")
        print("Feita a ordenacao em sort..;"),
    )
).start()

File: PO1508/Ordenacao/Principal.cs
using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

public class Principal
{
    public static void Main()
    {
        List<int> lista = new List<int>();
        List<int> listaBolha = new List<int>();

        GeradorNumeros.GerarNumerosLista(lista, 1000);
        GeradorNumeros.GerarNumerosLista(listaBolha, 1000);
        
            Console.WriteLine("Ordenando a lista de números com o algoritmo de bolha ...");
            Ordenacao.Bolha(listaBolha);
            Console.WriteLine("Feita a ordenação com o algoritmo de bolha.");


        
            Console.WriteLine("Ordenando a lista de números com o método Sort ...");
            lista.Sort();
            Console.WriteLine("Feita a ordenação com o método Sort.");
      
    }
}
